qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -333,6 +333,7 @@
             no RHEL.6, RHEL.7
             gagent_check_type = os_basic_info
             cmd_get_host_name = hostname
+            cmd_set_host_name = "hostnamectl set-hostname newhostname; cat /etc/hostname"
```

Comment:
if it's better to use replace ';' with '&&'
cmd_set_host_name = "hostnamectl set-hostname newhostname && cat /etc/hostname"